package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "npm run build -w web-local",
     "dev": "sh -c 'npm run dev-runtime & npm run dev-web & wait'",
-    "dev-web": "npm run dev -w web-local",
+    "dev-web": "npm run dev -w web-local & npm run check:watch -w web-local",
```

Comment:
Does this actually work? `&` sends the command to background. `&&` will wait for the previous to finish before running the next.